### PR TITLE
add user subodh101 as ci viewer and org member

### PR DESCRIPTION
**Pull request by subodh101**
What?
As per the title.

Why?
For the ci debugging purposes.

### CODE REVIEW

File: google_groups/ci-viewer.members.txt [hunk 1]
```diff
@@ -11,3 +11,4 @@ iman.tabrizian@gmail.com
 nrchakradhar379@gmail.com
 redhat-team@kubeflow.org
 yliu989@bloomberg.net
+subodh.pushkar@gmail.com
```
Comment (jlewi) [inline google_groups/ci-viewer.members.txt:14]: alphabetical please.

**Comment by jlewi** [comment]
You might as well go ahead and join the GitHub org in this PR as well so you can trigger tests.

**Comment by subodh101** [comment]
Thanks! made the changes.

**Comment by subodh101** [comment]
Let's get this in if everything looks good.

**Comment by jlewi** [comment]
/lgtm
/approve

apologies for the delay

**Comment by jlewi** [comment]
Note the group sync is not automatic. It will probably be a couple days as I'm working on automating the process.

**Pull request merged**